### PR TITLE
Add GeoparquetLoader

### LINKED ISSUE #54: Add GeoparquetLoader
Loads geoparquet file (or files)

### PR DESCRIPTION
**Pull request by RaczeQ**
Closes #54

### CODE REVIEW

File: srai/loaders/geoparquet_loader.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    def load(
+        self,
+        file_path: Union[Path, str],
+        index_column: Optional[str] = None,
+        columns: Optional[List[str]] = None,
+        area: Optional[gpd.GeoDataFrame] = None,
+    ) -> gpd.GeoDataFrame:
+        """
+        Loads a geoparquet file.
```
Comment (Calychas) [inline srai/loaders/geoparquet_loader.py:34]: use imperative mode -> "Loads a geoparquet file."

**Review by Calychas — CHANGES_REQUESTED** [review]
Great work! There are couple of issues that need to be addressed

File: pyproject.toml [hunk 2]
```diff
@@ -102,7 +103,7 @@ add-ignore = ["D212"]
 syntax = 'google'
 black = true
 recursive = true
-docstring-length = [60, 100]
+docstring-length = [40, 80]
```
[the remark this answers is not in the document]
Comment (RaczeQ) [reply pyproject.toml:106]: Had issues with automatic line breaking in `docformatter` and flake8 still was triggering too long line errors. But this change didn't fix the issue anyway 😞

File: srai/loaders/geoparquet_loader.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+        Args:
+            file_path (Union[Path, str]): parquet file path.
+            index_column (Optional[str], optional): Column that will be used as an index.
+                If not provided, automatic indexing will be applied by default. Defaults to None.
+            columns (Optional[List[str]], optional): List of columns to load.
+                If not provided, all will be loaded. Defaults to None.
+            area (Optional[gpd.GeoDataFrame], optional): Mask to clip loaded data.
+                If not provided, unaltered data will be returned. Defaults to None.
```
[the remark this answers is not in the document]
Comment (RaczeQ) [reply srai/loaders/geoparquet_loader.py:43]: This form was automatically generated by the `autodocstring` extension. I can change it based on the `google` format definition.

File: pyproject.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (RaczeQ) [reply pyproject.toml:106]: https://github.com/PyCQA/docformatter/issues/68
Seems like we cannot break lines automatically for now. Will disable breaking and we will do it manually.

File: srai/loaders/geoparquet_loader.py (hunk 1 above)
Comment (RaczeQ) [reply srai/loaders/geoparquet_loader.py]: Changed to `Load a geoparquet file.`

Comment (Calychas) [reply srai/loaders/geoparquet_loader.py]: Great! Sorry for misspelling

**Review by Calychas — APPROVED** [review]

**Pull request merged**